public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -55,15 +55,18 @@ export function panelMenuBehavior(menu: VizPanelMenu, isRepeat = false) {
       return;
     }
 
-    items.push({
-      text: t('panel.header-menu.view', `View`),
-      iconClassName: 'eye',
-      shortcut: 'v',
-      onClick: () => DashboardInteractions.panelMenuItemClicked('view'),
-      href: getViewPanelUrl(panel),
-    });
+    const editPanel = Boolean(dashboard.state.editPanel);
```

Comment:
```suggestion
    const isEditingPanel = Boolean(dashboard.state.editPanel);
```

Makes it a bit more clear what your checking / caring about